### ISSUE TITLE
Right after clicking navigator buttons, they will be triggerd by spacebar

### ISSUE THREAD

**Issue by FyisFe** [slidevjs/slidev#36]
## Problem: 
Clicking any buttons in the navigation bar, then press the spacebar. It will trigger the "next animation" event and the event associated with the button together.

For example, I change to the dark mode at first:
<img src = "https://user-images.githubusercontent.com/59299011/117544761-71a0bb80-b055-11eb-949b-a388a825147d.png" width = "600px">

Then I press the spacebar:
<img src = "https://user-images.githubusercontent.com/59299011/117544825-c04e5580-b055-11eb-962c-652dcac10da9.png" width = "600px">

## Reason:
I believe it is because the button gets focused after clicked, then pressing the spacebar can trigger the button. If I click somewhere to make it lose focus, the problem will not happen. 

## Solution:
I am not sure if it is a problem that needs to be solved, but I think it may ruin the user experience since they have to click somewhere else after clicking the navigator buttons. 

I have solved it by ```blur()```.  If you think it is worthwhile to do so, I can submit a PR later. 
 
<b>Finally, thanks for this awesome project again! 👍 <b>

**Comment by antfu** [comment]
Yeah, nice catch. I am not sure how to handle this properly. Maybe let's see your approach in PR. Thanks!
